src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -22,46 +22,50 @@
     private Clipper clipper = new Clipper(new Coordinate(0, 0), new Coordinate(0, 0));
     private Dinghy dinghy = new Dinghy(new Coordinate(0, 0), new Coordinate(0, 0));
 
-    private AircraftCarrier computer_aircraftCarrier = new AircraftCarrier(new Coordinate(2, 2), new Coordinate(2, 7));
-    private Battleship computer_battleship = new Battleship(new Coordinate(2, 8), new Coordinate(6, 8));
-    private Submarine computer_submarine = new Submarine(new Coordinate(4, 1), new Coordinate(4, 4));
-    private Clipper computer_clipper = new Clipper(new Coordinate(7, 3), new Coordinate(7, 5));
-    private Dinghy computer_dinghy = new Dinghy(new Coordinate(9, 6), new Coordinate(9, 6));
+    private AircraftCarrier computer_aircraftCarrier = new AircraftCarrier(new Coordinate(0, 0), new Coordinate(0, 0));
+    private Battleship computer_battleship = new Battleship(new Coordinate(0, 0), new Coordinate(0, 0));
+    private Submarine computer_submarine = new Submarine(new Coordinate(0, 0), new Coordinate(0, 0));
+    private Clipper computer_clipper = new Clipper(new Coordinate(0, 0), new Coordinate(0, 0));
+    private Dinghy computer_dinghy = new Dinghy(new Coordinate(0, 0), new Coordinate(0, 0));
```

Comment:
Initialize computer ships here, just like player ships in BattleshipModel class.